Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
+     * Setting to false will hide mosaic button for
+     * the admin screen.
+     */
+    public function setIsShowMosaicButton($isShow)
+    {
+        $var = (bool) $isShow;
+
+        if (!$var) {
+            $this->hideMosaicButton();
+
+            return;
+        }
+
+        $this->showMosaicButton();
```

Comment:
The whole method could be summarized like this : `return ((bool) $showButton ? $this->showMosaicButton() : $this->hideMosaicButton();`